packages/image-to-png/src/index.ts

```diff
@@ -124,14 +128,22 @@ export class ImageToPng {
       });
       // TODO: Catch errors
       stream.on('end', () => {
-        resolve(pngBase64);
+        let buffer = Buffer.from(pngBase64, 'base64');
```

Comment:
This is not necessary. Are the PNGs converted `color type` is not 2 at some point?